sw/device/lib/testing/test_rom/BUILD

```diff
@@ -77,10 +77,10 @@ cc_library(
     # This should be built only for english breakfast and skipped if using wildcards.
     tags = ["manual"],
     deps = [
-        "//hw/ip/gpio/data:gpio_c_regs",
-        "//hw/ip/otp_ctrl/data:otp_ctrl_c_regs",
-        "//hw/ip/spi_device/data:spi_device_c_regs",
-        "//hw/top_earlgrey/ip_autogen/flash_ctrl:flash_ctrl_c_regs",
+        "//hw/top:flash_ctrl_c_regs",
```

Comment:
Can we generate the test_rom_libs in a loop over tops?

Reply:
The `test_rom` situation is a bit of mess: the englishbreakfast test_rom is very hacky and doesn't use the same files as earlgrey (e.g. it stubs out some some functions related to non-existent IPs) which makes this a very manual process. I think we will need to revisit how the `test_rom` is built and try to avoid having one `cc_library` per top if possible.